### DISCUSSION TITLE
VSCode Intelephense

### DISCUSSION THREAD

**Discussion by MtDalPizzol** [pestphp/pest#530] [Q&A]
Hi folks. I'm getting started with Pest and I'm using VSCode, but it keeps highlighting global functions as undeifned. Is there a way to make VSCode stop squeaking for things like "it", "uses" and "expect"?

**Comment by nunomaduro** [comment]
I've you consider using PHPStorm instead? We've a plugin for PHPStorm that works great.

**Comment by MtDalPizzol** [comment]
Hi, @nunomaduro . Thanks for the quick response, but PHPStorm is not an option for me at the moment. =/
